src/main/java/com/microsoft/graph/models/AppCredentialRestrictionType.java

```diff
@@ -7,40 +7,36 @@
 
 
 /**
- * The Enum Lobby Bypass Scope.
```

Comment:
Just to follow up here.
The fix was applied via https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/7663994?_a=files

Fix can be confirmed on staging metadata. But is yet to propagate fully on prod. ETA is latest tomorrow. 

cc @ramsessanchez @baywet @Ndiritu @SilasKenneth